mcs/tools/linker/Makefile

```diff
@@ -7,6 +7,8 @@ PROGRAM = monolinker.exe
 LIB_REFS = System System.Core System.Xml Mono.Cecil
 
 TEST_CASES := \
+	link-all/link-all.cs \
```

Comment:
Let's not add these tests in this form

Reply:
so just build/link them, but not run the actual tests right?